R/viz.R

```diff
@@ -658,180 +810,341 @@ pretty_lollipop_plot = function(maf_df,
 #' #get ashm regions
 #' some_regions = grch37_ashm_regions
 #'
-#' get_mutation_frequency_bin_matrix(these_samples_metadata = dlbcl_bl_meta,
-#'                                   regions_df = some_regions)
-#'
-get_mutation_frequency_bin_matrix = function(regions,
-                                             regions_df,
-                                             these_samples_metadata,
-                                             seq_type = "genome",
-                                             region_padding = 1000,
-                                             metadataColumns = c("pathology"),
-                                             sortByColumns = c("pathology"),
-                                             expressionColumns = c(),
-                                             orientation = "sample_rows",
-                                             skip_regions = c("MYC", "BCL2", "IGLL5"),
-                                             customColour = NULL,
-                                             slide_by = 100,
-                                             window_size = 500,
-                                             min_count_per_bin = 3,
-                                             min_bin_recurrence = 5,
-                                             min_bin_patient = 0,
-                                             region_fontsize = 8,
-                                             cluster_rows_heatmap = FALSE,
-                                             cluster_cols_heatmap = FALSE,
-                                             show_gene_colours = FALSE,
-                                             legend_row = 3,
-                                             legend_col = 3,
-                                             legend_direction = "horizontal",
-                                             legendFontSize = 10,
-                                             from_indexed_flatfile = TRUE,
-                                             mode = "slms-3"){
-
-  if(missing(regions)){
-    if(missing(regions_df)){
-      regions_df = grch37_ashm_regions #drop MYC and BCL2
-      regions_df = grch37_ashm_regions %>%
-        dplyr::filter(!gene %in% skip_regions)
+#' mut_count_matrix <- get_mutation_frequency_bin_matrix(
```

Comment:
Example is out of date given the new function name

Reply:
Good catch